miso-web/src/main/webapp/scripts/run_ajax.js

```diff
@@ -511,14 +511,15 @@ Run.ui = {
   },
 
   addContainerByBarcode: function (runId) {
-    jQuery('#addRunNoteDialog')
-      .html("<form><p>Adding a container will lose any unsaved changes!</p>" +
+    jQuery('#addContainerDialog')
+      .html("<form><p>Adding a container now will lose any unsaved changes to the run information above!</p>" +
+            "<p>If there are no unsaved changes or you wish to continue anyway, enter the container's barcode below.</p>" +
```

Comment:
Concerned that this is too wordy and may serve to frighten the user into believing they can/are making a mistake. The previous warning is concise and says it all.